src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Shared.xml

```diff
@@ -0,0 +1,63 @@
+<linker>
+  <assembly fullname="System.Private.CoreLib">
+    <type fullname="Interop/Globalization">
+      <!-- Internal API used by tests only. -->
+      <method name="GetICUVersion" />
```

Comment:
Separate from this PR, if this is only used by tests, can it be part of _LibraryBuild file instead?

Reply:
I looked at this yesterday, 

https://github.com/dotnet/runtime/blob/a9626b9d3e1f52234c6864cc5d9f11252fa5d3e4/src/libraries/Common/src/Interop/Interop.ICU.cs#L27-L28

I was thinking of just moving the P/Invoke to the tests.

But now that I look again, our Globalization.Native shim is replaced with QCall -- can you do that from an assembly outside of S.P.Corelib?

If not, then yes it could be moved to `_LibraryBuild`.

Reply:
QCalls are for CoreLib-only.